### DISCUSSION TITLE
Welcome to HTML Discussions!

### DISCUSSION THREAD

**Discussion by MAKWritingHouse** [MAKWritingHouse/HTML#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by murilobarbosaa** [comment]
Hello!

**Reply by MAKWritingHouse** [reply]
Hello there! **Welcome** to our _community Discussions_! It's _fantastic_ to have you here. 😊 We encourage you to ask any **questions** you may have, share your thoughts and ideas, and actively engage with other community members. We believe in fostering a welcoming and open-minded environment where everyone's contributions are valued.

To kick things off, why not _introduce yourself_ to the **community**? We'd love to hear about who you are and what brings you here. Feel free to share your experiences and what you hope to achieve by being a part of this **wonderful community**. We're all here to support each other and build something great _together_! 💪
